frontend/resourceadm/components/ResourceDeployEnvCard/ResourceDeployEnvCard.tsx

```diff
@@ -40,42 +32,52 @@ export const ResourceDeployEnvCard = ({
   envName,
   currentEnvVersion,
   newEnvVersion,
+  onClick,
+  loading,
 }: ResourceDeployEnvCardProps): React.ReactNode => {
   const { t } = useTranslation();
 
-  const handleOnClick = () => {
-    console.log('Coming soon...');
-  };
-
   return (
     <div className={classes.cardWrapper}>
-      <Paragraph size='small' className={classes.envName}>
-        <strong>{envName}</strong>
-      </Paragraph>
-      <div className={classes.envWrapper}>
-        <Tag color='neutral' variant='outlined' size='small'>
-          v{currentEnvVersion}
-        </Tag>
-        {newEnvVersion && (
-          <>
-            <div className={classes.arrowWrapper}>
-              <ArrowRightIcon title={t('resourceadm.deploy_card_arrow_icon', { env: envName })} fontSize='1.5rem' />
-            </div>
-            <Tag color='success' variant='outlined' size='small'>
-              v{newEnvVersion}
+      {loading ? (
+        <Spinner title={t('resourceadm.deploy_deploying')}></Spinner>
+      ) : (
+        <>
+          <Paragraph size='small' className={classes.envName}>
+            <strong>{envName}</strong>
+          </Paragraph>
+          <Paragraph size='small' className={classes.versionNumberText}>
+            {t('resourceadm.deploy_version_number_text')}
+          </Paragraph>
+          <div className={classes.envWrapper}>
+            <Tag color='neutral' variant='outlined' size='small'>
+              {currentEnvVersion}
             </Tag>
-          </>
-        )}
-      </div>
-      <div className={classes.buttonWrapper}>
-        <Button
-          aria-disabled={!isDeployPossible}
-          onClick={() => isDeployPossible ? handleOnClick() : undefined}
-          size='small'
-        >
-          {t('resourceadm.deploy_card_publish', { env: envName })}
-        </Button>
-      </div>
+            {newEnvVersion && (
+              <>
+                <div className={classes.arrowWrapper}>
+                  <ArrowRightIcon
+                    title={t('resourceadm.deploy_card_arrow_icon', { env: envName })}
+                    fontSize='1.5rem'
+                  />
+                </div>
+                <Tag color='success' variant='outlined' size='small'>
+                  {newEnvVersion}
+                </Tag>
+              </>
+            )}
+          </div>
+          <div className={classes.buttonWrapper}>
+            <Button
```

Comment:
Dersom isDeployPossible === false, hva opplever brukeren når vedkommende trykker på Publiser? Sånn jeg leser koden så er knappen fortsatt aktivert, men onClick er undefined. Vil ikke det bety at brukeren opplever at knappen ikke gjør noe? Burde vi legge til en validering eller melding som forteller brukeren hvorfor deploy ikke er mulig? 🤔

Reply:
Endrer til å bruke ```disabled``` i steden for. 
<br />
Ang. feilmelding, så er det alltid en alert på toppen av siden som informerer om statusen, eksempel: 
<img width="770" alt="Screenshot 2023-10-16 at 08 50 31" src="https://github.com/Altinn/altinn-studio/assets/133344438/8ef0a9d8-5453-45db-a620-3a5741037b05">